### ISSUE TITLE
Improve STS references

### ISSUE THREAD

**Issue by pluralitybook** [pluralitybook/plurality#23]
Footnote 24 alludes to the Science and Technology Studies literature with some citations, but they may be biased and poorly chosen.  Improvement is welcome.

**Comment by pluralitybook-dao** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 12-Jan-24 22:28:59 UTC` by Gov4Git `dev`

### Notice `x2nn4f`
Started managing this issue as Gov4Git concern `23` with initial __priority score__ of `0.000000`.

This project is managed by [Gov4Git](https://gov4git.org), a decentralized governance system for collaborative git projects.
To participate in governance, __install the [Gov4Git desktop app](https://github.com/gov4git/desktop-application)__.

**Comment by pluralitybook-dao** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 01-Mar-24 21:15:49 UTC` by Gov4Git `dev`

#### Notice `qb4uv6`
This issue's __priority score__ is now `150.000000`.
The __cost of priority__ is `22500.000000`.
The __projected bounty__ is now `22500.000000`.

#### Notice `zjhx6t`
The set of eligible proposals claiming this issue is empty.

**Comment by Fossj117** [comment]
(Some open-ended thoughts in case helpful; can take a pass at integrating this if it seems like reasonable direction)

Assuming this is regarding this footnote: https://github.com/pluralitybook/plurality/blob/ac582317c84f62a945ca52f5219dadc98171ec7d/contents/english/02-00-information-technology-and-democracy-a-widening-gulf.md?plain=1#L234

Which is cited here in the book: 

> But science fiction writers are not alone.  The primary theme of the field of Science and Technology Studies (STS), including the philosophy, sociology, and history of science, has been the contingency and possibility inherent in the development of science and technology and the lack of any single necessary direction for their evolution[^STS]. 

If we want to connect this more to STS / philosophy of tech world, I think a key term is "technological determinism" (science/tech proceed according to their own necessary logic & have a unidirectional effect on society). That is the viewpoint being criticized here. 

I'm not deeply familiar with all of the references in the footnote, but certainly a classic STS cite on this in the context of technology is: 

* Pinch, T. J., & Bijker, W. E. (1984). The social construction of facts and artefacts: Or how the sociology of science and the sociology of technology might benefit each other. Social studies of science, 14(3), 399-441. 

This piece is also in the Bijker, Hughes, & Pinch anthology already cited in the footnote; but should probably cite this specific article, which established the 'social construction of technology' (SCOT) paradigm? ([wiki](https://en.wikipedia.org/wiki/Social_construction_of_technology) has a decent summary). Winner "Do Artifacts Have Politics?" is also a classic on how social forces shape the direction of technological development. 

* Winner, L. (2017). Do artifacts have politics?. In Computer ethics (pp. 177-192). Routledge.

Could also potentially cite Bloor who helped establish the "strong program" in the sociology of knowledge which informed SCOT (more about knowledge/science than technology): 

* Bloor, D. (1991). Knowledge and social imagery. University of Chicago Press.

Various ideas/cites from Sheila Jasanoff could fit in here as well. Her idea of "co-production" is basically a suggested analytic alternative to tech determinist frame. A cite discussing that: 

* Jasanoff, S. (Ed.). (2004). States of knowledge: the co-production of science and the social order. Routledge.

There's also some good discussion in the intro of: 

* Jasanoff, S. (2016). The ethics of invention: Technology and the human future. WW Norton & Company.

Jasanoff & Kim's concept of a "sociotechnical imaginary" also seems potentially relevant to some of the book's framing:

> [sociotechnical imaginaries are] collectively held, institutionally stabilized, and publicly performed visions of desirable futures, animated by shared understandings of forms of social life and social order attainable through, and supportive of, advances in science and technology

From this piece (PDF online [here](https://law.unimelb.edu.au/__data/assets/pdf_file/0009/3305673/1.-Jasanoff-and-Kim-2015-Dreamscapes-of-Modernity-Sociotechnical-Imaginari.pdf)):

* Jasanoff, S. (2015). Future imperfect: Science, technology, and the imaginations of modernity. Dreamscapes of modernity: Sociotechnical imaginaries and the fabrication of power, 1-33.

That definition is a bit wordy, but basically just this idea of: collectively held visions of desirable futures that are posed as reachable through technoscientific development, and which shape present choices about the direction of technological development. 

Seems like the book has moved away from talking about "Abundance Technocracy" (AT) and "Entrepreneurial Sovereignty" (ES) explicitly (and Plurality as an alternative), but I was basically thinking of these as examples of sociotechnical imaginaries.
